uwrt_mars_rover_hw/src/uwrt_mars_rover_hw_drivetrain.cpp

```diff
@@ -0,0 +1,77 @@
+#include "uwrt_mars_rover_hw/uwrt_mars_rover_hw_drivetrain.h"
+
+namespace uwrt_mars_rover_hw {
+
+UWRTRoverHWDrivetrain::UWRTRoverHWDrivetrain() : UWRTRoverHWDrivetrain("UWRTRoverHWDrivetrain") {}
+
+bool UWRTRoverHWDrivetrain::init(ros::NodeHandle& root_nh, ros::NodeHandle& robot_hw_nh) {
+  bool param_fetched = robot_hw_nh.getParam("joints", joint_names_);
+  if (!param_fetched) {
```

Comment:
should return false if this fails?

Reply:
maybe, havent thought to much about it. it probably should

Reply:
will fix in #39